PDV/TSF Drip

AthrunZala33092 opened the pull request:
<!-- Guidelines: https://docs.spacestation14.io/en/getting-started/pr-guideline -->

## About the PR
<!-- What did you change? --> This is my first PR, I am trying to add drip. Specifically additional TSF uniforms along with a cap, mantle, and cloak for the Vizier. (Also I'm hoping that my test deletion/readding Sevas is not in this, cause it wanted me to merge all commits weirdly.
<!-- If this is a code change, summarize at high level how your new code works. This makes it easier to review. -->

## Why / Balance
<!-- Discuss how this would affect game balance or explain why it was changed. Link any relevant discussions or issues. --> Just a QOL change.

## How to test
<!-- Describe the way it can be tested --> Should be visible on loadout.

## Media
<!-- Attach media if the PR makes ingame changes (clothing, items, features, etc). 
Small fixes/refactors are exempt. Media may be used in SS14 progress reports with credit. -->

## Requirements
<!-- Confirm the following by placing an X in the brackets [X]: -->
- [X ] I have read and am following the [Pull Request and Changelog Guidelines](https://docs.spacestation14.com/en/general-development/codebase-info/pull-request-guidelines.html).
- [X ] I have added media to this PR or it does not require an ingame showcase.
- [X ] I can confirm this PR contains no AI-generated content, and did not use any AI-generated content.
<!-- You should understand that not following the above may get your PR closed at maintainer’s discretion -->

## Breaking changes
<!-- List any breaking changes, including namespaces, public class/method/field changes, prototype renames; and provide instructions for fixing them.
This will be posted in #codebase-changes. -->

**Changelog**
<!-- Add a Changelog entry to make players aware of new features or changes that could affect gameplay.
Make sure to read the guidelines and take this Changelog template out of the comment block in order for it to show up.
Changelog must have a :cl: symbol, so the bot recognizes the changes and adds them to the game's changelog. -->

:cl:
- add: New Drip: PDV Vizier's hat, cloak, and mantle, along with TSF dress uniform options.

AthrunZala33092 committed:
testcommit

AthrunZala33092 committed:
testcommit

AthrunZala33092 committed:
Adding TSF uniforms

Hope this works!

AthrunZala33092 committed:
Actually made the TSFMC uniforms an entity

Fixing the entity error

AthrunZala33092 committed:
Adding PDV drip now

Added cap, mantle, and cloak for the Vizier.

monolith8319 committed:
chore: automatically update REUSE headers

AthrunZala33092 committed:
Update meta.json

Removed the monkey mention

AthrunZala33092 committed:
Update meta.json

Removed monkey mention

DieselMohawkTheSequel commented on Resources/Prototypes/Entities/Clothing/Head/hats.yml:
Item name should not be capitalized.
There is an ' before the description.

DieselMohawkTheSequel commented on Resources/Prototypes/_Mono/Entities/Clothing/Neck/cloaks.yml:
Item name should not be capitalized.

DieselMohawkTheSequel reviewed (commented):
Looking good, just some minor inconsistencies, good work.

AthrunZala33092 committed:
Update hats.yml

Removed the ' and changed to lower case.

AthrunZala33092 committed:
Update cloaks.yml

Changed names to lowercase as requested

OneZerooo0 reviewed (changes requested):
Its a vibe, but the TSFMC officer/enlisted uniforms are way too similar - either remove enlisted and let them use officer uniforms, or make the enlisted more distinct

AthrunZala33092 committed:
Update jumpsuits.yml

Removing the enlisted uniform

AthrunZala33092 committed:
Update universal_groups.yml

Removing the enlisted uniform

AthrunZala33092 commented:
> Its a vibe, but the TSFMC officer/enlisted uniforms are way too similar - either remove enlisted and let them use officer uniforms, or make the enlisted more distinct

I'm not sure how to make it more distinct, so for now I've just edited the entity/loadout files to remove the enlisted uniform. My intention had been to separate the two between the enlisted and officer roles, but I didn't know how to do that and both ended up getting it. They should all be able to use the officer uniform now, however I'm unsure how to delete the enlisted folder with the sprites from the github in the PR.

thoughtlessuser commented:
> > Its a vibe, but the TSFMC officer/enlisted uniforms are way too similar - either remove enlisted and let them use officer uniforms, or make the enlisted more distinct
> 
> I'm not sure how to make it more distinct, so for now I've just edited the entity/loadout files to remove the enlisted uniform. My intention had been to separate the two between the enlisted and officer roles, but I didn't know how to do that and both ended up getting it. They should all be able to use the officer uniform now, however I'm unsure how to delete the enlisted folder with the sprites from the github in the PR.

Try adding small things that represent their ranking
The higher ranks have medals and talismens and tassles 

Dont worry if theyre not detailed enough, the most youd be adding is like 6 pixels in their chest area or on their shoulder

no need to be super differential but give players a clear visual interpretation of what rank a person is by looking at the suit, militaries uses ranking medals and shit like i said

AthrunZala33092 commented:
> > > Its a vibe, but the TSFMC officer/enlisted uniforms are way too similar - either remove enlisted and let them use officer uniforms, or make the enlisted more distinct
> > 
> > 
> > I'm not sure how to make it more distinct, so for now I've just edited the entity/loadout files to remove the enlisted uniform. My intention had been to separate the two between the enlisted and officer roles, but I didn't know how to do that and both ended up getting it. They should all be able to use the officer uniform now, however I'm unsure how to delete the enlisted folder with the sprites from the github in the PR.
> 
> Try adding small things that represent their ranking The higher ranks have medals and talismens and tassles
> 
> Dont worry if theyre not detailed enough, the most youd be adding is like 6 pixels in their chest area or on their shoulder
> 
> no need to be super differential but give players a clear visual interpretation of what rank a person is by looking at the suit, militaries uses ranking medals and shit like i said

Alright, well I'll just leave it as the officer variant for now and visit the sprites in the future to see if additional ones can be added/modified for more drip and figuring out how to set it to individual loadouts if needed.

thoughtlessuser commented:
> > > > Its a vibe, but the TSFMC officer/enlisted uniforms are way too similar - either remove enlisted and let them use officer uniforms, or make the enlisted more distinct
> > > 
> > > 
> > > I'm not sure how to make it more distinct, so for now I've just edited the entity/loadout files to remove the enlisted uniform. My intention had been to separate the two between the enlisted and officer roles, but I didn't know how to do that and both ended up getting it. They should all be able to use the officer uniform now, however I'm unsure how to delete the enlisted folder with the sprites from the github in the PR.
> > 
> > 
> > Try adding small things that represent their ranking The higher ranks have medals and talismens and tassles
> > Dont worry if theyre not detailed enough, the most youd be adding is like 6 pixels in their chest area or on their shoulder
> > no need to be super differential but give players a clear visual interpretation of what rank a person is by looking at the suit, militaries uses ranking medals and shit like i said
> 
> Alright, well I'll just leave it as the officer variant for now and visit the sprites in the future to see if additional ones can be added/modified for more drip and figuring out how to set it to individual loadouts if needed.

try looking at captains gear for help with ideas on ranking decals and tassles n shit, theres alot on that

AthrunZala33092 committed:
Update cloaks.yml

Changed Shah to Sultan as Arctic has decided on that being the PDV leader for his lore.

OneZerooo0 commented:
Still waiting on the officer suit changes

AthrunZala33092 commented:
> Still waiting on the officer suit changes

EDIT: Ahem. Realized I hadn't renamed the description/label at all, so I went in there and changed the entity/loadout label and all that. Had to delete the enlisted folder and rename the officer folder locally, I think I did this right?

AthrunZala33092 committed:
Trying to rename officer/remove enlisted here

Trying this to delete the enlisted uniform and rename the officer one.

AthrunZala33092 committed:
Merge branch 'testbranch' of https://github.com/AthrunZala33092/Monolith into testbranch

AthrunZala33092 committed:
Accidental merge error
